Code/RDGeneral/RDValue-doublemagic.h

```diff
@@ -411,6 +410,22 @@ inline bool rdvalue_cast<bool>(RDValue v) {
   throw boost::bad_any_cast();
 }
 
+// The RDValue has no getKey implementation
+//  so implement our own
+struct KeyIntPair {
+  int key;
+  RDValue val;
+  KeyIntPair() : key(), val() {}
+  KeyIntPair(int k, const RDValue &v) : key(k), val(v) {}
+  KeyIntPair(const std::string &s, RDValue_cast_t v) :
+  key(tagmap.get(s)), val(v) {}
+  
+  inline void setKey(int k) { key = k; }
+  inline int  getKey() const { return key; }
+
+  static RDTags tagmap;
```

Comment:
How much can/does RDKit do at link-time? If KeyIntPair::tagmap is ever accessed at link-time this will become link-time order problem. Really should be a Meyer's singlton: 

```
static RDTags &getTagMap();
```